docs_devel/docs/90.ReleaseProcedure.md

```diff
@@ -7,50 +7,87 @@ See [Versioning](05.Versioning.md) for details about a versioning scheme.
 ## 1. Pre-announce on an omegat-dev mailing list
 
 Especially if there is new translatable material, give the L10N team some lead
-time to update translations.
+time to update translations. It is better to announce the due date
+at the start of the release engineering, Step 3, instead of the planned
+release date.
 
-## 2. Decide bundled JREs versions
+email template:
+
+```text
+Subject: release plan for OmegaT 6.x
+
+Hi all.
+
+The planed changes for releasing 6.x are now all merged,
+and latest weekly release looks good for the general release.
+ 
+We are now starting the release process.
+Please check a latest release procedure at 
+https://omegat.readthedocs.io/en/latest/90.ReleaseProcedure/
+
+Due dates will be as follows:
+
+- Feature freeze: today
+
+- Documentation updating phase
+    - Manual content freeze:  14, Dec.
+    - UI Localisation freeze: 20, Dec.
+    - Manual localisation freeze: 20, Dec.
+
+- Update JRE versions to up-to-date: 20, Dec.
+
+- Last weekly release for the candidate: 21, Dec.
+
+- Check and confirmation
+    - Ask confirmation for the latest weekly as the genral release 
+
+- Release build and announce: 24, Dec.
+    - git tag the source
+    - trigger CI/CD for the release
+    - Update web site download links
+    - Send announce to users ML.
+
+```
+
+## 2. Choose the bundled JREs and update their major versions
 
 See if an update is available for the bundled JREs,
 and update `ci/azure-pipelines/release_steps.yml` if necessary.
 
+If you change a bundled JREs version, you must also change `build.gradle` lines 276-282,
```

Comment:
Are those the right line numbers?

They seem to be in lines 434 to 438 in the current `build.gradle` file in the 6.1 master branch. Maybe it would be better to say something like "search for the section starting with "def macJRE in `build.gradle`", or add a comment in the file to to identify the section containing the file name, and have that be the search target, so that the developer manual isn't affected by updates to `build.gradle` that change the line numbers.

Reply:
Good catch!  I would like to `def` to very first of `build.gradle` and add a title for the block, and update manual to reference to title.

Reply:
@Kazephil could you check it again?

Reply:
Looks much better. I just committed a few more minor wording tweaks, so I think this one is good to go unless someone spots a typo in my tweaked phrases.